fails to build on i686

```
   Compiling libmacchina v7.1.0 (/home/joey/Downloads/libmacchina)
error[E0308]: mismatched types
   --> src/shared/mod.rs:259:42
    |
259 |         let disk_size = stats.f_blocks * stats.f_bsize as UInt;
    |                                          ^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0277]: cannot multiply `u64` by `u32`
   --> src/shared/mod.rs:259:40
    |
259 |         let disk_size = stats.f_blocks * stats.f_bsize as UInt;
    |                                        ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Mul<Rhs>`:
              <&'a u64 as Mul<u64>>
              <&u64 as Mul<&u64>>
              <u64 as Mul<&u64>>
              <u64 as Mul>

error[E0308]: mismatched types
   --> src/shared/mod.rs:262:37
    |
262 |         let used_byte = disk_size - free;
    |                                     ^^^^ expected `u64`, found `u32`

error[E0277]: cannot subtract `u32` from `u64`
   --> src/shared/mod.rs:262:35
    |
262 |         let used_byte = disk_size - free;
    |                                   ^ no implementation for `u64 - u32`
    |
    = help: the trait `Sub<u32>` is not implemented for `u64`
    = help: the following other types implement trait `Sub<Rhs>`:
              <&'a u64 as Sub<u64>>
              <&u64 as Sub<&u64>>
              <u64 as Sub<&u64>>
              <u64 as Sub>

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `libmacchina` due to 4 previous errors
```

--- a/src/shared/mod.rs
+++ b/src/shared/mod.rs
@@ -256,7 +256,7 @@ pub(crate) fn disk_space(path: String) -> Result<(u64, u64), ReadoutError> {
 
         let stats: libc::statfs = unsafe { s.assume_init() };
 
-        let disk_size = stats.f_blocks * stats.f_bsize as UInt;
+        let disk_size = stats.f_blocks as UInt * stats.f_bsize as UInt;
         let free = stats.f_bavail as UInt * stats.f_bsize as UInt;
 
         let used_byte = disk_size - free;

